src/pages/index.js

```diff
@@ -1,13 +1,42 @@
 import React from 'react'
 import Link from 'gatsby-link'
 
-const IndexPage = () => (
-  <div>
-    <h1>Hi people</h1>
-    <p>Welcome to your new Gatsby site.</p>
-    <p>Now go build something great.</p>
-    <Link to="/page-2/">Go to page 2</Link>
-  </div>
-)
+export default ({ data }) => {
+  console.log(data)
+  return (
+    <div>
+      <h1>Projects</h1>
+      <h4>{data.allMarkdownRemark.edges.length} Posts</h4>
```

Comment:
I like the pattern you used in floods with setting a nicely named variable from the data object. I think this would be a great place to do that.